wrench/src/yaml_frame_writer.rs

```diff
@@ -324,40 +324,50 @@ impl YamlFrameWriter {
         let bytes = data.bytes.take().unwrap();
         let (path_file, path) = Self::next_rsrc_paths(&self.rsrc_prefix, &mut self.next_rsrc_num, &self.rsrc_base, "img", "png");
 
-        let ok = match data.format {
+        // takes a buffer with a stride and copies it into a new buffer that has stride == width
+        fn unstride<T: Clone>(mut slice: &[T], width: usize, stride: usize) -> Vec<T> {
+            let mut result = Vec::new();
```

Comment:
Oh, thought this was on WR itself, if it's in the recording code probably doesn't matter that much.